### ISSUE TITLE
List of JSON data is not populating as expected

### ISSUE THREAD

**Issue by Alihassanc5** [bhch/django-jsonform#172]
My model structure is:

```
ITEMS_SCHEMA = {
     'type': 'dict',
     'keys': {
        'id': {'type': 'integer'},
        'answer': {'type': 'string'},
        'question': {'type': 'string'},
        'required': {'type': 'boolean'},
        'help_text': {'type': 'string'},
        'identifier': {'type': 'string'},
     },
     'required': ['id', 'answer', 'question']
}


class ResearchReport(TimeStampedModel):
    questions = ArrayField(JSONField(schema=ITEMS_SCHEMA), help_text="Questions of the research report", null=True, blank=True)    
```
  And it's showing like this on admin panel:
  
<img width="714" alt="image" src="https://github.com/bhch/django-jsonform/assets/62539376/635f008c-ada7-4b00-b3cd-3a3f17308689">

**Comment by Alihassanc5** [comment]
@bhch could you please help me? What am I doing wrong?

**Comment by bhch** [comment]
Hi, thanks for reporting this issue.

Currently, the `ArrayField` doesn't support nesting `JSONField`. I've fixed this issue now. I'll release a new version soon.

**Comment by Alihassanc5** [comment]
@bhch thank you for quick support, could you please tell me the expected date of new release?

**Comment by bhch** [comment]
A couple of days.

**Comment by bhch** [comment]
Released in [v2.23.0](https://github.com/bhch/django-jsonform/releases/tag/v2.23.0).
